codegenerator/cli/templates/static/codegen/src/db/DbFunctionsEntities.res

```diff
@@ -103,21 +103,38 @@ external whereEqQuery: (
   ~value: Js.Json.t,
 ) => promise<Js.Json.t> = "whereEqQuery"
 
-let makeWhereEq = (
-  type entity,
-  sql: Postgres.sql,
+@module("./DbFunctionsImplementation.js")
+external whereGtQuery: (
+  ~table: Table.table,
+  ~sql: Postgres.sql,
+  ~fieldName: string,
+  ~value: Js.Json.t,
+) => promise<Js.Json.t> = "whereGtQuery"
+
+let makeWhereQuery = (type entity, sql: Postgres.sql) => async (
+  ~operator: TableIndices.Operator.t,
   ~entityMod: module(Entities.Entity with type t = entity),
-) => async (
   ~fieldName: string,
   ~fieldValue: 'fieldValue,
   ~fieldValueSchema: S.t<'fieldValue>,
   ~logger=Logging.logger,
 ): array<entity> => {
   let module(Entity) = entityMod
+
+  let queryType = switch operator {
+  | Eq => "whereEq"
+  | Gt => "whereGt"
+  }
```

Comment:
string coersian could work fine here to remove an additional switch. Not a problem though